consensus/parlia/parlia.go

```diff
@@ -1441,10 +1508,19 @@ func (p *Parlia) Delay(chain consensus.ChainReader, header *types.Header, leftOv
 		delay = delay - *leftOver
 	}
 
-	// The blocking time should be no more than half of period
-	half := time.Duration(p.config.Period) * time.Second / 2
-	if delay > half {
-		delay = half
+	// The blocking time should be no more than half of period when snap.TurnLength == 1
+	timeForMining := time.Duration(p.config.Period) * time.Second / 2
+	if snap.TurnLength > 1 {
+		if snap.lastBlockInOneTurn(header.Number.Uint64()) {
+			// To ensure that the next validator produces and broadcasts blocks in a timely manner,
+			// set the value of timeForMining to a small amount
+			timeForMining = 500 * time.Millisecond
+		} else {
+			timeForMining = time.Duration(p.config.Period) * time.Second * 2 / 3
```

Comment:
why 2/3, to reserve some buffer?

Reply:
> why 2/3, to reserve some buffer?
yes.
the final value need to be decided after benchmark,
but if we set it to 3 seconds, the network will not stable